plans/network/pingpong.go

```diff
@@ -64,8 +57,9 @@ func pingpong(runenv *runtime.RunEnv) error {
 	ipC := byte((seq >> 8) + 1)
 	ipD := byte(seq)
 
-	config.IPv4 = &runenv.TestSubnet.IPNet
+	config.IPv4 = runenv.TestSubnet
 	config.IPv4.IP = append(config.IPv4.IP[0:2:2], ipC, ipD)
+	config.IPv4.Mask = []byte{255, 255, 255, 255}
```

Comment:
Is this correct?

Reply:
I think yes, because the ping-pong is requesting specific IPs. @hacdias that's correct, right?